src/client/search/categories/CategoryFilters.js

```diff
@@ -1,26 +1,104 @@
 import React, { Component } from 'react';
 import YearSelector from './YearSelector';
 import { connect } from 'react-redux';
+import { newPromiseChain } from './../../../shared/utilities';
+import LeagueCategory from './LeagueCategory';
+import TeamCategory from './TeamCategory';
+import PlayerCategory from './PlayerCategory';
+import {
+  fetchAllFootballLeagueTeams, fetchFootballTeamPlayers, removeSelectedFootballTeamPlayers
+} from './categoryFilterActions';
+import { addSearchTerm } from './../searchActions';
 import moment from 'moment';
 
 class CategoryFilters extends Component {
   componentDidMount() {
-    $('.menu .item').tab();
-  }
+    $('.menu .item').tab({
+      onVisible: (tabPath) => {
+        // animation for the league counts on the tabs
+        const currentYear = moment().year() - 1;
+
+        // for each year tab
+        for (let y = currentYear; y >= 2013; y--) {
```

Comment:
Make 2013 a const explaining WHY it's 2013 - e.g. like EARLIEST_YEAR_AVAILABLE_FROM_FOOTBALL_API or whatever